lib/config/bigquery.go

```diff
@@ -5,11 +5,12 @@ import "fmt"
 type BigQuery struct {
 	// PathToCredentials is _optional_ if you have GOOGLE_APPLICATION_CREDENTIALS set as an env var
 	// Links to credentials: https://cloud.google.com/docs/authentication/application-default-credentials#GAC
-	PathToCredentials string `yaml:"pathToCredentials"`
-	DefaultDataset    string `yaml:"defaultDataset"`
-	ProjectID         string `yaml:"projectID"`
-	Location          string `yaml:"location"`
-	BatchSize         int    `yaml:"batchSize"`
+	PathToCredentials  string `yaml:"pathToCredentials"`
+	DefaultDataset     string `yaml:"defaultDataset"`
+	ProjectID          string `yaml:"projectID"`
+	Location           string `yaml:"location"`
+	BatchSize          int    `yaml:"batchSize"`
+	UseStorageWriteAPI bool   `yaml:"__useStorageWriteAPI"` // Not officially supported yet.
```

Comment:
For config settings that are not intended for production use I think it's nice to prefix them with "__" to indicate that the are not officially supported, that way we can get rid of them without breaking backwards compatibility and having to bump the version.

Reply:
Sure